tests/storage/storage.py

```diff
@@ -33,7 +33,7 @@ def cold_migration_then_come_back(vm: VM, prov_host: Host, dest_host: Host, dest
 
     if integrity_check:
         # the vdi will be destroyed with the vm
-        vdi = prov_sr.create_vdi(virtual_size=1 * GiB)
+        vdi = prov_sr.create_vdi(None, virtual_size=1 * GiB)
```

Comment:
IMHO it bring more confusion to reader than a default param, if you want to be really explicit may you add repeat name_label:

Reply:
It's symmetrical with calls that do have a name_label, and if the reader is intrigued by the parameter, at least this informs them about the fact that there is such a parameter. But if there are concerns, we can also abandon this positional argument idea. It's not a big deal.